ouroboros-network/src/Ouroboros/Network/Diffusion.hs

```diff
@@ -66,21 +66,26 @@ data ExtraTracers (p2p :: P2P) extraState extraDebugState extraFlags extraPeers
 
 -- | Diffusion arguments which depend on p2p mode.
 --
+-- TODO: do we need both `exception` and `resolverError`?
```

Comment:
exception if I am not mistaken is for accomodating the `BootstrapPeersCriticalTimeoutError`